.gitignore

```diff
@@ -1,44 +1,37 @@
-.DS_Store
-.bundle
-public/system/*
-db/*.sqlite3
-*.sqlite3
+*.sql
 *.sqlite
-log/*.*
-tmp/**/*
-vendor/bundle/**
+*.sqlite3
 .DS_Store
+.bundle
+.ruby-version
+.yarn-integrity
 /.DS_Store
-update
-config/deploy.rb
-config/deploy/*
+/node_modules
+/public/packs
+/public/packs-test
+/yarn-error.log
 config/alonetone.yml
-config/skylight.yml
-config/scout_apm.yml
+config/certs/*
 config/config.yml
 config/database.yml
 config/defensio.yml
-config/newrelic.yml
-config/memcached.yml
+config/deploy.rb
 config/deploy/*
-config/certs/*
-*.sql
+config/memcached.yml
+config/newrelic.yml
+config/scout_apm.yml
+config/skylight.yml
+db/*.sqlite3
+log/*.*
 public/mp3
 public/pics
+public/sitemap.xml
 public/source_files
 public/system
-public/sitemap.xml
-.ruby-version
-/public/packs
-/public/packs-test
-/node_modules
+public/system/*
+stats.json/public/packs
```

Comment:
I think we just want `stats.json` here — this is used for https://github.com/sudara/alonetone/issues/193